src/domains/sign/00_resource_groups.tf

```diff
@@ -18,3 +18,10 @@ resource "azurerm_resource_group" "sec_rg" {
 
   tags = var.tags
 }
+
+resource "azurerm_resource_group" "integration_rg" {
```

Comment:
why integration? (what's the usage of this rg?)

may we add a description?

Reply:
why not?

Reply:
In my opinion the appropriate place whether to explain the necessity of a resource group for the event hub, is the README of the terraform module https://github.com/pagopa/terraform-azurerm-v3/tree/main/eventhub#how-to-use-it

In this way everyone could benefit from this info

Reply:
I mean an inline comment like:

> needed to integrate Firma con IO with external domains, products or platforms (ie. eventhub for billing, ...)

Reply:
Done